Tests/DependencyInjection/Compiler/AddDependencyCallsCompilerPassTest.php

```diff
@@ -189,6 +189,32 @@ function ($element) {
     /**
      * @covers Sonata\AdminBundle\DependencyInjection\Compiler\AddDependencyCallsCompilerPass::process
```

Comment:
Don't know. I don't really like the `@covers` usage because it override the code coverage check.

With this tag, I can say "I cover toto" with a test of "tata". :-)